src/components/SubHeader/subHeader.css

```diff
@@ -1,20 +1,25 @@
 
 /* sub header css start */
-.appSubHeader {
+.subHeader_text {
     display: flex;
     flex-direction: column;
     align-items: center;
-    padding: 50px 0px;
+    padding: 50px;
```

Comment:
when i said `padding: 50px 0px` is not necessary.
i didn't mean `padding: 50px 0` won't work. the `px` after `0` just wasn't necessary. that's what i meant.
> in case that's the reason you took off the 0px completely.